cmd/root.go

```diff
@@ -170,20 +170,25 @@ func init() {
 
 // initConfig reads in config file and ENV variables if set.
 func initConfig() {
+	initViper(viper.GetViper())
+}
+func initViper(v *viper.Viper) {
 	if cfgFile != "" { // enable ability to specify config file via flag
-		viper.SetConfigFile(cfgFile)
```

Comment:
Ah, we _do_ have that code, near https://github.com/mook-as/fissile/blob/ebd1ad311ea40b474688182d5ce9babd88d163e0/builder/packages_image.go#L165